charts/studio/templates/raycluster-studio.yaml

```diff
@@ -0,0 +1,70 @@
+{{- if .Values.ray.enabled -}}
+apiVersion: ray.io/v1alpha1
+kind: RayCluster
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: {{ .Release.Name }}-ray
+spec:
+  rayVersion: {{ .Values.ray.version }}
+  enableInTreeAutoscaling: true
+  autoscalerOptions:
+    upscalingMode: Default
+    idleTimeoutSeconds: 60
+    imagePullPolicy: IfNotPresent
+    resources:
+      resources:
+        cpu: 500m
+        memory: 500M
+      resources:
+        cpu: 500m
+        memory: 500M
+  headGroupSpec:
+    rayStartParams:
+      dashboard-host: 0.0.0.0
```

Comment:
Don't we want to expose it through Ingress?

Proposition, not blocker

Reply:
No (?) 😅 It's a service without any kind of authentication, and should only be available inside the cluster.

If anything, we could add a route on Studio for the Ray dashboard in the future, but always behind some kind of authentication mechanism.

Reply:
No problem then! I also thought it was some web UI